src/screens/Search.js

```diff
@@ -1,20 +1,30 @@
 import { render } from '../shared/dom.js';
 import { TilesGrid } from '../components';
 import ApiProvider from '../providers/ApiProvider.js';
-import ConsoleLogger from '../utils/ConsoleLogger.js';
+import { SearchForm } from '../components/Search/Form.js';
 
 const template = `
 <div id="search-screen">
+    <template id="search-form"></template>
     <template id="tiles-grid"></template>
 </div>
 `;
 
 export const SearchScreen = async ({ renderOn }) => {
     render({ on: renderOn, html: template });
 
-    const movies = (await ApiProvider.search('Fast & Furious')).results;
+    const params = new URLSearchParams(window.location.search);
```

Comment:
We could standardize that and make a utility function from that, it would be helpfull cuz we would get the category from params as well on category screen